hexa/plugins/connector_s3/api.py

```diff
@@ -141,9 +146,19 @@ def generate_sts_user_s3_credentials(
         aws_secret_access_key=principal_credentials.secret_access_key,
     )
 
+    role_session_name = f"sts-{principal_credentials.username}-{session_identifier}"
+
+    if len(role_session_name) >= 63:
```

Comment:
We need to be able to identify the user. Can we use the user id instead?

It think passing the user id as the `session_identifier` would do the trick.

Reply:
(the need is for auditing purpose)